File: CodingSamples/DotNet/Fundamentals/Basics/BuiltinTypesTest/DemoApp/Investment.cs
class Investment
{
    //a member declared with 'public' keyword is visible outside its class
    //a method declared with 'static' keyword must be called on the class
    //using . operatoor
    public static double FutureValue(double installment, int years, bool risk)
    {
        double i = risk ? 0.08 : 0.06;
        return (installment / i) * (System.Math.Pow(1 + i, years) - 1);
    }
}

File: CodingSamples/Java/Fundamentals/Basics/PrimitiveTypesTest/demo/Investment.java
class Investment {
    
    //a method declared with public keyword is visible outside of its class,
    //a method declared with static keyword can be called on the class itself
    public static double futureValue(double installment, int years, boolean risk) {
        double i = risk ? 0.08 : 0.06;
        return (installment / i) * (Math.pow(1 + i, years) - 1);
    }
}
